### ISSUE TITLE
Add rocm-smi-lib to Prerequisites Section of readme?

### ISSUE THREAD

**Issue by v4u6h4n** [aristocratos/btop#710]
Hey there :-) Been waiting for this release for a while, thanks for all the work on it.

GPU monitoring didn't work initially until I looked at the logs and installed rocm-smi-lib. It might be a good idea to add it as a dependency for GPU support, along with whatever nvidia counterpart(s) exist, in the Prerequisites section of the readme, especially to avoid pointless bug submissions from people that aren't going to look at their logs before posting.

(Installed from Arch repo)

**Comment by aristocratos** [comment]
@v4u6h4n 
See https://github.com/aristocratos/btop#gpu-compatibility

**Comment by v4u6h4n** [comment]
@aristocratos 

I did initally see that section, however I ignored it as I just installing via pacman from the Arch repo, and the GPU dependency was listed was under the "Compilation Linux" section, and I assumed that it was specific to building from source only.

It personally doesn't make a difference to me, but I'm just thinking it might save you and others some time responding to many "GPU not working" issues from less experienced Linux users.

Here's a [pull request](https://github.com/aristocratos/btop/pull/711), but understand if it doesnt merit changing.
